x-pack/platform/test/onechat_api_integration/apis/spaces.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { ListToolsResponse } from '@kbn/onechat-plugin/common/http_api/tools';
+import type { ListAgentResponse } from '@kbn/onechat-plugin/common/http_api/agents';
+import { AGENT_BUILDER_ENABLED_SETTING_ID } from '@kbn/management-settings-ids';
+import type { FtrProviderContext } from '../../api_integration/ftr_provider_context';
+import { spaceUrl } from '../utils/spaces';
+import { createTool, deleteTool } from '../utils/tools';
+import { createAgent, deleteAgent } from '../utils/agents';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const spaces = getService('spaces');
+  const kibanaServer = getService('kibanaServer');
+  const es = getService('es');
+
+  describe('Space support', () => {
```

Comment:
suggestion: add a testcase demonstrating that having a tool without the space field defined is returned in the default space?

Reply:
Yeah good idea, but I can't use our APIs for that, I will need to directly use the ES client.